src/main/java/core/basesyntax/service/operation/BalanceHandler.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.service.operation;
+
+import core.basesyntax.Main;
+
+public class BalanceHandler implements OperationHandler {
+    @Override
+    public void operation(String fruitName, int fruitQuantity) {
```

Comment:
```suggestion
    public void process(String fruitName, int fruitQuantity) {
```
better to use verbs for method names